src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -960,8 +982,7 @@ internal static IntPtr GetCachedSortHandle(string sortName)
             }
         }
 
-        // See https://github.com/dotnet/coreclr/blob/master/src/utilcode/util_nodependencies.cpp#L970
-        private static readonly bool[] s_highCharTable = new bool[0x80]
+        private static ReadOnlySpan<bool> HighCharTable => new bool[0x80]
```

Comment:
I don't realize this worked for anything other than byte/sbyte. I guess it works for bool since that's a byte in size as well and there's no endianness issue?

Reply:
Yes, this works for all 1-byte primitive types: https://github.com/dotnet/roslyn/pull/24621/files#diff-99b8cc443e7f403bed9c694c22fa6c2eR378